content/en/apps/features/targets/_index.md

```diff
@@ -6,40 +6,55 @@ description: >
 
-Counts without a goal display a simple black number count. Counts with a goal display the value of the goal on the right side and a colored count (green if the count is above the goal, or red if the count is below the goal).
+A count without a goal displays a simple black number count. A count with a goal displays the value of the goal on the right side and a colored count in the center indicating progress towards achieving the goal. Progress is displayed in green if the count is equal to or above the goal, or in red if the count is below the goal.
 
 ### Percent Widgets
 
-Percentage widgets provide insight into how much data matches a specific criteria against data that does not. This is calculated based on a true / false statement. This is often configured to represent accomplishment. For example, newborns should be visited within the first three days of life (“true”) can be displayed next to newborns that were not visited within the first three days (“false”).
+{{< figure src="mobile-percent.png" link="mobile-percent.png" class="right col-6 col-lg-3" >}}
 
-Next to the percentage, the count of reports used in the calculation are shown (e.g. 16 of 20 [newborn visits] on-time). CHWs have found this helpful in interpreting this information. 
+Percent widgets display a ratio, which helps to provide insight into the proportion that matches a defined criteria. For example, the proportion of newborns delivered in a facility can be presented as a percent with respect to all registered deliveries.
 
-An optional goal can be set, such as “70% of newborns should be visited within the first three days.” Conditional styling can be configured to show green if a goal has been met and red if the goal has not been met. 
+An optional goal can be set, such as “100% of patients with a fever should be given a malaria Rapid Diagnostic Test (mRDT),” to visualize progress towards achieving a target. Widget styling is configured to show green if the goal has been met and red if the goal has not been met. Next to the percent with a goal, the count of reports used in the calculation are shown (e.g. “16 of 20 with mRDT”). CHWs have found this helpful in interpreting target information.
+
+## Supervisor View
+
+Aggregate targets were introduced in v3.9 to provide Supervisors actionable information about their CHWs, and help with coaching and performance management. Aggregate targets combine the info for each CHW that a Supervisor manages. 
+
+{{< figure src="mobile-supervisor.png" link="mobile-supervisor.png" class="right col-4 col-lg-6" >}}
+
+Clicking on an aggregate widget opens the detailed view with the data for each individual CHW. If a CHW is performing below the target goal, their value will be highlighted in red, making it easier for Supervisors to know with which CHWs to follow up.
+
+<br />
+<br />
+<br />
+<br />
+<br />
```

Comment:
The note displays over the image. We could add more padding to the note, though it's not a consistent problem so it may have to be conditional if we want to code it in